doc/DEVELOPERS.md

```diff
@@ -0,0 +1,256 @@
+# Get Start
+
+for Fedora 38
+
+## Install PostgreSql
+
+    sudo dnf install postgresql-server
+
+for Fedora 38, this will install postgresql 15
+
+## Install pgmoneta
+
+### pre-install
+
+    sudo dnf install git gcc cmake make libev libev-devel openssl openssl-devel systemd systemd-devel zlib zlib-devel libzstd libzstd-devel lz4 lz4-devel libssh libssh-devel libcurl libcurl-devel python3-docutils libatomic bzip2 bzip2-devel libarchive libarchive-devel cjson cjson-devel
+
+### Choose what suits you, if above didn't work
+
+    sudo dnf install cmake
+    sudo dnf install bzip2-devel
+    sudo dnf install lz4-devel
+    sudo dnf install libev-devel
+    sudo dnf install libarchive-devel
+    sudo dnf install cjson cjson-devel
+    sudo dnf install python3-docutils
+    sudo dnf install libssh-devel
+    sudo dnf install libcurl-devel
+    sudo dnf install systemd-devel
+
+### build
+
+    cd /usr/local
+    sudo git clone https://github.com/pgmoneta/pgmoneta.git
+    cd pgmoneta
+    sudo mkdir build
+    cd build
+    sudo cmake -DCMAKE_INSTALL_PREFIX=/usr/local ..
+    sudo make
+    sudo make install
+
+### Check version
+
+    pmoneta -?
+
+If you see an error saying `error while loading shared libraries: libpgmoneta.so.0: cannot open shared object` running the above command. you may need to locate where your `libpgmoneta.so.0` is. It could be in `/usr/local/lib` or `/usr/local/lib64` depending on your environment. Add the corresponding directory into `/etc/ld.so.conf`, or alternatively, create a file called `pgmoneta_shared_library.conf` under `/etc/ld.so.conf.d/`, and add your directory into it. Remember to run `ldconfig` to make the change effective
```

Comment:
I think this need a rewrite because standard development you would do

```
./pgmoneta
```

in the `build/src` directory...

Reply:
`cd /usr/local`
I used this command to install it in `/usr/local`. 
After this, we can directly use pgmoneta, as the PATH will be able to find it.
If necessary, I will delete this and modify it.

Reply:
I think you should write it as, either the developer will use `./pgmoneta` in `build/src`, or install into `/usr/local/` and show how `/usr/local/lib/` and `/usr/local/lib64` are enabled for `ld.so.conf`